packages/core/components/DropZone/styles.module.css

```diff
@@ -1,38 +1,38 @@
 .DropZone {
-  margin-left: auto;
-  margin-right: auto;
+  --drop-animation-ms: 250ms;
+  --resize-animation-ms: 150ms;
+
   position: relative;
-  min-height: 100%;
+  height: 100%;
+  min-height: var(--min-empty-height);
   outline-offset: -2px;
   width: 100%;
 }
 
-.DropZone-content {
-  min-height: 128px;
-  height: 100%;
+.DropZone--hasChildren {
+  min-height: 0;
```

Comment:
Hm, the minimum height causes issues because it makes assumptions about the user's UI that are incorrect. This results in a preview UI that doesn't match the end state.

The implementation now only expands the DropZone if it's empty, otherwise it will snap to the smallest item it contains.

Reply:
Yeah totally but if you drag a small item like text as the first item then it's fairly tricky to get the next item as the dropZone is now tiny target maybe, It might be more accessible if it was always a large area to drag to.